modules/luci-base/luasrc/util.lua

```diff
@@ -640,15 +640,15 @@ function checklib(fullpathexe, wantedlib)
 	local fs = require "nixio.fs"
 	local haveldd = fs.access('/usr/bin/ldd')
 	if not haveldd then
-		return -1
+		return false
 	end
 	local libs = exec("/usr/bin/ldd " .. fullpathexe)
 	if not libs then
-		return 0
+		return false
 	end
 	for k, v in ipairs(split(libs)) do
 		if v:find(wantedlib) then
-			return 1
+			return true
 		end
 	end
 	return 0
```

Comment:
I've pushed the fix for the missed return 0 that is now return false.
Thanks for the review; it probably would have cost me some time to
discover the cause of the problems that would have caused for my testing.

Regards,

Daniel

On 16-04-08 02:11 AM, Hannu Nyman wrote:
> In modules/luci-base/luasrc/util.lua
> <https://github.com/openwrt/luci/pull/698#discussion_r58983729>:
> 
>>  	end
>>  	for k, v in ipairs(split(libs)) do
>>  		if v:find(wantedlib) then
>> -			return 1
>> +			return true
>>  		end
>>  	end
>>  	return 0
> 
> Shouldn't this be false instead of 0 ?????
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openwrt/luci/pull/698/files/8b5ffa3f88b0fe0d5a77fc2a704b22aacf946fd0#r58983729>
>